rclcpp_lifecycle/test/test_lifecycle_node.cpp

```diff
@@ -504,6 +565,19 @@ TEST_F(TestDefaultStateMachine, test_graph) {
     topic_names_and_types["/testnode/transition_event"][0].c_str(),
     "lifecycle_msgs/msg/TransitionEvent");
 
+  EXPECT_LE(1u, test_node->count_publishers("/testnode/transition_event"));
```

Comment:
Why not testing the exact value?

Reply:
I thought I needed this hack to pass the test. It does not look like it's the case anymore. Fixed